db/agents.yml

```diff
@@ -39,7 +39,11 @@ agents:
     name: null
     type: Mobile App
     os: Android
-  - regex: ^Breaker.+CFNetwork
+  - regex: ^Bose/
+    name: Bose
+    type: Smart Home
+    os: Amazon OS #???
```

Comment:
I see some posts indicating Alexa support on Bose smart speakers.  I'm assuming this is the OS, but really I have no idea.